src/main/java/frc/robot/subsystems/SubsystemCondition.java

```diff
@@ -0,0 +1,79 @@
+package frc.robot.subsystems;
+
+import java.util.EnumSet;
+
+public class SubsystemCondition {
+    
+    /**
+     * A list of the possible options conditions can have
+     */
+    public enum ConditionOptions {

```

Comment:
How about `State`?
Then when invoking this elsewhere it will look like `SubsystemOperation.State.TELEOP`